kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/test/java/org/kie/server/services/jbpm/ui/form/render/FormRendererTest.java

```diff
@@ -242,12 +245,16 @@ public void testRenderOfBasicTaskFormWithSelectRadioGroupAndData() {
         inputs.put("radio", "radio2");
         inputs.put("decimal", 123.5);
         inputs.put("hwSpec_", "name####123####111####id");
+        inputs.put("startDate_", LocalDateTime.of(LocalDate.of(2020, 12, 1), LocalTime.now()));
```

Comment:
Why not to use LocalTime.of(hour, minutes) and then check the timestamp as a whole? If these are Local data types I think timezones etc won't affect the test execution.

Reply:
Because the PR opened in the customer is using a LocalDateTime